.github/workflows/build-images.yml

```
@@ -21,6 +21,12 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
+      - name: Lint Dockerfile
+        uses: hadolint/hadolint-action@v3.1.0
+        with:
+          dockerfile: ${{ matrix.distribution }}/Dockerfile
+          config: "hadolint.yml"
```

Can it be moved into the .github dir? 

```suggestion
          config: ".github/hadolint.yml"
```